aptos-move/aptos-vm-types/src/storage.rs

```diff
@@ -69,14 +69,14 @@ impl StoragePricingV1 {
         }
 
         match op {
-            Creation(data) | CreationWithMetadata { data, .. } => {
+            Creation(value_size) => {
```

Comment:
Would it make sense to match or map on the op.write_len() here? (since that's what determines the logic at the moment).

Reply:
here at least the logic is different whetehr it is creation (additional write_data_per_new_item) or not